doc/guide/theory.rst

```diff
@@ -0,0 +1,169 @@
+
+For particles without spherical symmetry, the decoupling approximation
+is more accurate, with
+
+.. math::
+
+    I(Q) = n [\langle F F^* \rangle
+        + \langle F \rangle \langle F \rangle^* (S(Q) - 1)]
+
+Or equivalently,
+
+.. math:: I(Q) = P(Q)[1 + \beta\,(S(Q) - 1)]
+
+with the form factor $P(Q) = n \langle F F^* \rangle$ and
+$\beta = \langle F \rangle \langle F \rangle^* \big/ \langle F F^* \rangle$.
+These approximations can be extended to heterogeneous systems using averages
+over size, $\langle \cdot \rangle_\mathbf\xi = \int_\Xi P_\mathbf\xi \langle\cdot\rangle\,\mathrm d\mathbf\xi \big/ \int_\Xi P_\mathbf\xi \,\mathrm d\mathbf\xi$ and setting
+$n = V_f\big/\langle V \rangle_\mathbf\xi$.
+
+Further improvements can be made using the local monodisperse
+approximation (LMA) or using partial structure factors as done in [#bressler]_,
+but these are not implemented in this code.
+
+For hollow shapes, *volfraction* is computed from the material in the
+shell rather than the shell plus solvent inside the shell.  Using
+$V_e(\mathbf\xi)$ as the enclosed volume of the shell plus solvent and
+$V_c(\mathbf\xi)$ as the core volume of solvent inside the shell, we
+can compute the average enclosed and shell volumes as
+
+.. math::
+    :nowrap:
+
+    \begin{align*}
+    \langle V_e \rangle &= \frac{
+        \int_\Xi P_\mathbf\xi V_e(\mathbf\xi)\,\mathrm d\mathbf\xi
+    }{ \int_\Xi P_\mathbf\xi\,\mathrm d\mathbf \xi } \\
+    \langle V_s \rangle &= \frac{
+        \int_\Xi P_\mathbf\xi (V_e(\mathbf\xi) - V_c(\mathbf\xi))\,\mathrm d\mathbf\xi
+    }{ \int_\Xi P_\mathbf\xi\,\mathrm d\mathbf \xi }
+    \end{align*}
+
+Given $n$ particles and a total solvent volume $V_\text{out}$ outside the
+shells, the volume fraction of the shell, $\phi_s$ and the shell plus
+enclosed solvent $\phi_e$ are
+
+.. math::
+    :nowrap:
+
+    \begin{align*}
+    \phi_s &= \frac{n \langle V_s \rangle}{n \langle V_s \rangle + n \langle V_c \rangle + V_\text{out}}
+           = \frac{n \langle V_s \rangle}{V_\text{total}} \\
+    \phi_e &= \frac{n \langle V_e \rangle}{n \langle V_e \rangle + V_\text{out}}
+           = \frac{n \langle V_e \rangle}{V_\text{total}}
+    \end{align*}
+
+Dividing gives
+
+.. math::
+
+    \frac{\phi_S}{\phi_P} = \frac{\langle V_e \rangle}{\langle V_s \rangle}
+
+so the enclosed volume fraction can be computed from the shell volume fraction
+and the form:shell volume ratio as
+
+.. math::
+
+    \phi_S = \phi_P \langle V_e \rangle \big/ \langle V_s \rangle
+
+.. note::
+
+    Prior to Sasmodels v1.0.5 (Nov 2020), the intermediate $P(Q)$ returned by
+    the interaction calculator did not incorporate the volume normalization and
+    so $I(Q) \ne P(Q) S(Q)$. This became apparent when $P(Q)$ and $I(Q)$ were
+    plotted together. Further details can be found `here <https://github.com/SasView/sasview/issues/1698>`_.
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
```

Comment:
I wonder if we want pull the relevant bits of that thread in here? When we move from github to gitlab we may not have access to old issues.

Reply:
So we had a discussion of this at the Code Camp. There is a good point being made here, but the question is how best to address it without over-burdening the docs (the feeling was the current Note told most readers all they needed to know) or going to a lot of effort in the absence of a strong motivation (there are no plans or proposals on the table to move to GitLab). What is in place at the moment covers both bases. There was a suggestion to pdf the GitHub issue and offer it as a download, though this would only be a snapshot of the issue at a point in time, of course.
What I propose we do is leave the current text as is, but I will create an Admin ticket to check the docs for GitHub issue links before any move to GitLab.